File: parameter_estimation/parameters.m
function [t0,Tf,t,h,m,a,b,c,e,f,j,r1,r2,alpha,beta,a_T,a_N,a_L,a_C,gamm] = parameters()

% Initial and final times
t0 = 0;
Tf = 30;

% Grid parameters
m = 99;
t = linspace(t0,Tf,m+1);
h = t(2)-t(1);

% Known parameter values
a = 2.31e-1;
b = 2.146e-10;
c = 5.156e-14;
f = 1e-2;
e = 1/9*f;
j = 1.245e-4;
r1 = 5.156e-12;
r2 = 1e-15;
beta = 6.3e-3;
alpha = 3e9*beta;

% Functional parameter values
a_T = 1;
a_N = 1;
a_L = 1;
a_C = 1;

% Regularization parameter values
gamm = 0.00000000001;

File: parameter_estimation_PYTHON/parameters.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Sat Feb 13 12:26:29 2021

@author: susanthkakarla
"""

import numpy as np
import math

def parameters():
    
    a = 0.231;
    b = 0.021;
    c = 3*pow(10,-9);
    e = 0.22;
    f = 0.01;
    j = 1.245*pow(10,-4);
    m = 5*pow(10,-3);
    r1 = 61.6*pow(10,-7)*9;
    r2 = 2.4*pow(10,-7)*9;
    alpha = 1.62;
    beta = 6.3*pow(10,-3);
    
    
    # No. of time interval points
    Nt = 200;
    t = np.linspace(0, 20, num = Nt, endpoint=True)
    step = t[2]-t[1];
    step = (1-math.exp(-50*step))/50

    # Regularization parameter
    nu = 0.001;
    
    return a, b, c, e, f, j, m, r1, r2, alpha, beta, t, Nt, step, nu
